packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -170,9 +170,55 @@ curl -X POST \
 
 > **Note**: You can copy the entire [response from the previous `GET /api/v1/apps/${id}/credentials/keys` request](#response-example) in to the `jwks.keys` array. The `created`, `lastUpdated`, and `expiresAt` properties are ignored in the POST request. Alternatively, you can remove them from the `jwks.keys` parameter body.
 
+#### Assign admin roles to the OAuth 2.0 service app
+
+> **Note:** Before the **Assign admin roles to public client apps** feature, the Super Administrator (`SUPER_ADMIN`) role was automatically assigned to service apps when they're created. You can now fine-tune the resources that a service app can access by assigning specific standard or custom admin roles. Ensure that the **Assign admin roles to public client apps** Self-Service feature is enabled in your org for granular role assignment. See [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information on the Okta Self-Service Feature Manager.
```

Comment:
In this note, can we callout that with **Assign admin roles to public client apps**, the role is no longer assigned and needs to be assigned before proceeding further?

Reply:
ok, i'll add this point.